src/spring/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 Release History
 ===============
+1.2.0
```

Comment:
May I ask why you upgraded the Minor version (1.2.0) instead of Patch version (1.1.6) ?

Reply:
Updated to 1.1.6.